packages/node_modules/@webex/plugin-meetings/api.md

```diff
@@ -188,6 +192,7 @@ Get the PMR and add it as a property on this object.
 
 ### _end_
 - ending, ended
+> Do we need `ending`? This may be bad UX. If I want to end, it should end immediately.
```

Comment:
Ending still indicates a status that an end is in flight though. We do the same for `leaving` and `joining`

Reply:
Yeah, I suppose the UX is independent. Someone could choose to show that status or not.